x-pack/plugins/security_solution/common/api/asset_criticality/common.gen.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { z } from 'zod';
+
+/*
+ * NOTICE: Do not edit this file manually.
+ * This file is automatically generated by the OpenAPI Generator, @kbn/openapi-generator.
+ */
+
+export type CreateAssetCriticalityRecord = z.infer<typeof CreateAssetCriticalityRecord>;
+export const CreateAssetCriticalityRecord = z.object({
+  /**
+   * The ID value of the asset.
+   */
+  id_value: z.string(),
+  /**
+   * The field representing the ID.
+   */
+  id_field: z.string(),
+  /**
+   * The criticality level of the asset.
+   */
+  criticality_level: z.enum(['very_important', 'important', 'normal', 'not_important']),
+});
+
+export type AssetCriticalityRecord = z.infer<typeof AssetCriticalityRecord>;
+export const AssetCriticalityRecord = CreateAssetCriticalityRecord.and(
+  z.object({
+    /**
+     * The time the record was created or updated.
+     */
+    '@timestamp': z.string().datetime(),
```

Comment:
I would say I am not a big fan of `@timestamp` in the response body, its just not a nice key to work with in JS :D You have to quote it to access it from an object. I think I would probably prefer `created_at` and `updated_at`.

Reply:
I think in the original comment from @oatkiller we assume that API repspone will have @timestamp as created_at, and updated_at field for updated time.

Agree with you comment about UI, I think it's good idea to return from API `created_at` and `updated_at`. 

But not sure what the consequences if we don't have @timestamp field in the index

Reply:
@nkhristinin I think you are accidentally tagging someone called timestamp in your comment :D 

yeah I think `@timestamp` is only essential for datastreams so it may be we were intending to future proof?

Reply:
@hop-dev the fixation on `@timestamp` is due to our use of/adherence to [ECS](https://www.elastic.co/guide/en/ecs/current/ecs-base.html); it's a required field on all time-based documents. As Nikita mentioned, we use it as the de facto alternative to `created_at` in instances like this.

Reply:
> But not sure what the consequences if we don't have `@timestamp` field in the index

The consequence would be that our data isn't ECS compliant, which means it wouldn't work by default in a lot of places. Some features allow you to pick your timestamp field (e.g. data view creation, rule creation), but those are the exception. `@timestamp` is the convention, and IMO we would need a compelling reason to break from that.

P.S. I didn't see @oatkiller's comment so I might be missing some context here; apologies if that's the case.